crates/tauri-runtime-wry/src/monitor/mod.rs

```diff
@@ -17,16 +17,11 @@ mod macos;
 #[cfg(windows)]
 mod windows;
 
-pub struct PhysicalRect {
-  pub size: PhysicalSize<u32>,
-  pub position: PhysicalPosition<i32>,
-}
-
 pub trait MonitorExt {
   /// Get the work area of this monitor
   ///
   /// ## Platform-specific:
   ///
   /// - **Android / iOS**: Unsupported.
-  fn work_area(&self) -> PhysicalRect;
+  fn work_area(&self) -> Rect;
```

Comment:
I think it's time for a `PhysicalRect` 🙃

Reply:
yeah you are right, I have added them and added `LogicalRect` as well

Reply:
will see if I can upstream them later on.